Code/IO/include/sitkImageFileWriter.h

```diff
@@ -82,6 +82,23 @@ class SmartPointer;
       SITK_RETURN_SELF_TYPE_HEADER UseCompressionOff( void ) { return this->SetUseCompression(false); }
       /** @} */
 
+      /** \brief Set/Get name of ImageIO to use
+       *
+       * An option to override the automatically detected ImageIO used
+       * to write the image. The available ImageIOs are listed by the
+       * GetRegisteredImageIOs method. If the ImageIO can not be
+       * constructed an exception will be generated. If the ImageIO
+       * can not write the file an exception will be generated.
```

Comment:
The sentence "If the ImageIO can not write the file an exception will be generated." appears to be out of context here.